ui/pages/confirmations/confirmation/components/existing-requests-banner-alert/existing-requests-banner-alert.tsx

```diff
@@ -0,0 +1,51 @@
+import React, { useContext, useState } from 'react';
+import { useSelector } from 'react-redux';
+import {
+  MetaMetricsEventCategory,
+  MetaMetricsEventName,
+} from '../../../../../../shared/constants/metametrics';
+import {
+  BannerAlert,
+  BannerAlertSeverity,
+} from '../../../../../components/component-library';
+import { MetaMetricsContext } from '../../../../../contexts/metametrics';
+import { useI18nContext } from '../../../../../hooks/useI18nContext';
+import {
+  getQueuedRequestCount,
+  pendingConfirmationsSortedSelector,
+} from '../../../../../selectors';
+
+export const ExistingRequestsBannerAlert = () => {
+  const t = useI18nContext();
+  const trackEvent = useContext(MetaMetricsContext);
+
+  const [metricsSent, setMetricsSent] = useState(false);
+
+  const queuedRequestCount = useSelector(getQueuedRequestCount);
+  const showExistingRequestBannerAlert = queuedRequestCount > 0;
+
+  const pendingConfirmations = useSelector(pendingConfirmationsSortedSelector);
```

Comment:
Same here, are we interested in every approval type or only transactions and signatures?

Reply:
Same as https://github.com/MetaMask/metamask-extension/pull/25852#discussion_r1683088539